### DISCUSSION TITLE
Link Indexing

### DISCUSSION THREAD

**Discussion by lifeos-admin** [BeanieODM/beanie#346] [Question]
This might be a stupid question and this also might be happening in the background automatically as well since i am not that well versed with mongo, But are links possible to index and search with ? 
i.e. 
`
class Window(Document):
    name: str

class House(Document):
    window: Link(Window)
`

In this case, if i have a window document in memory and i would like to  search with `house.window.id==window.id`, is the window in House class automatically indexed ? I have tried using Index(Link) but it does not seem to work

**Comment by roman-right** [comment]
Hm. I have to check, how indexes against DBRef data type work in mongo. I'll let you know here the results.
